src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/dao/OcgmRepository.kt

```diff
@@ -20,7 +20,7 @@ class OcgmRepository : DataRepository<OcgmList>() {
     data.fileName = filename
     data.fileType = FileType.OCGM
     data.reset()
-    csvData.stream().filter { p: List<String>? -> data.index.getAndIncrement() > 0 }
+    csvData.stream().filter { data.index.getAndIncrement() > 0 }
```

Comment:
Most of these changes are improving the kotlin which was auto-generated from java